lib/teradata.js

```diff
@@ -17,6 +17,9 @@ function Teradata(config) {
   if (!config) throw new Error('Configuration required');
 
   this.connections = [];
+  config.jvmOptions = Array.isArray(config.jvmOptions)
+                      ? config.jvmOptions
+                      : ['-Xrs'];
   this.namedParamsRegex = /:([^\W]+)/g;
   this.config = _.defaultsDeep(config, {
     driver: './jars/',
```

Comment:
Any reason why we don't stick with the existing defaults pattern? Remove L20-L22, and add `jvmOptions: ['-Xrs']` to `_.defaultsDeep`:

```js
  this.config = _.defaultsDeep(config, {
    driver: './jars/',
    minPoolSize: 1,
    maxPoolSize: 100,
    keepalive: {
      interval: 60000,
      query: 'SELECT 1',
      enabled: false
    },
    logger: {
      level: 'error'
    },
    jvmOptions: ['-Xrs']
  });
```